### DISCUSSION TITLE
API Access

### DISCUSSION THREAD

**Discussion by gerardsyd** [TandoorRecipes/recipes#2354] [Q&A]
I think I'm being rather daft but how do I access the API? 

I'm trying to access via:
http://192.168.1.180:3250/api/shopping-list

and getting 
{
    "detail": "Authentication credentials were not provided."
}

Adding username and password as below:
http://192.168.1.180:3250/api/shopping-list/?username=XXXX&password=XXXX
doesn't seem to help either. 

I've also tried to create an auth token (via admin > tokens) and included in the header but that doesn't seem to work either. 

Any help much appreciated.

Thanks

**Comment by smilerz** [comment]
Authentication is token based.  You can find your token in settings.

**Reply by gerardsyd** [reply]
Thanks @smilerz - I was using the wrong auth token. Duh!

Quick other question. I'm trying to get my shopping lists from the api. When I hit the api at /api/shopping-list/

I get an empty array. The result is as below:
[ ]

But... in my shopping list, I have 20 items. 

Looking at the API explorer via the webpage, I am also still getting an empty array. 

Any ideas why this is not working? Per the documentation, using GET against that endpoint should give me the list of shopping lists with the UUID so I can then get items in each list...

Thanks!

**Reply by gerardsyd** [reply]
nvm, looks like the API endpoints have changed as per this post on reddit:
https://www.reddit.com/r/selfhosted/comments/10l8a7m/anyone_using_the_tandoor_api/

For anyone else looking, the endpoint api/shopping-list is no longer active and instead uses api/shopping-list-entry
